easybuild/easyconfigs/l/LAMMPS/LAMMPS-03Mar2020-CrayGNU-20.08-deepmd-1.2.1.eb

```diff
@@ -0,0 +1,45 @@
+# Contributed by TWR (CSCS)
+easyblock = 'MakeCp'
+
+name = 'LAMMPS'
+version = '03Mar20'
+_release = 'stable_3Mar2020'
+_deepmd_version = '1.2.1'
+versionsuffix = '-deepmd-%s' % _deepmd_version
+
+homepage = 'http://lammps.sandia.gov/'
+description = "LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) is a classical molecular dynamics simulation code designed to run efficiently on parallel computers."
+
+toolchain = {'name': 'CrayGNU', 'version': '20.08'}
+toolchainopts = { 'usempi': True, 'openmp': True }
+
+source_urls = ['https://github.com/lammps/lammps/archive']
+sources = ['%s.tar.gz' % (_release)]
+
+prebuildopts = ' cd ./src && '
+prebuildopts += ' cp -r $EBROOTDEEPMDMINKIT/USER-DEEPMD . && '
+prebuildopts += ' export CPATH=$EBROOTDEEPMDMINKIT/include/deepmd:$CPATH && '
+prebuildopts += ' make lib-plumed args="-p ${EBROOTPLUMED} -m runtime" && '
+prebuildopts += ' make yes-standard yes-user-cg-cmm yes-user-omp yes-user-reaxc yes-user-misc yes-user-plumed yes-user-deepmd && '
+prebuildopts += ' make no-latte no-voronoi no-reax no-poems no-meam no-kim no-kokkos no-mscg no-gpu no-message && '
+prebuildopts += ' make package-update && '
+prebuildopts += ' sed -e \'s/mpicxx/CC -fopenmp/\' ./MAKE/Makefile.mpi > ./MAKE/Makefile.omp && '
+prebuildopts += ' sed -i -e \'s/mpicxx/CC/\' ./MAKE/Makefile.mpi && '
+buildopts = [ ' mpi ', ' omp ' ]
+
+
+dependencies = [
+    ('cray-fftw', EXTERNAL_MODULE),
+    ('cray-python', EXTERNAL_MODULE),
+    ('deepmd-kit', _deepmd_version),
+    ('PLUMED', '2.5.1', '-mpi')
```

Comment:
I wonder why you need to have `PLUMED-2.5.1-mpi`, since `PLUMED-2.5.1` has MPI in it...

Reply:
I don't know. Should I rename it?

Reply:
Can you use the one that doesn't have the `-mpi` suffix? Does it work?